### ISSUE TITLE
Work with Hathifiles more closely

### ISSUE THREAD

**Issue by bmschmidt** [massivetexts/compare-tools#4]
Can we just load the metadata straight from hathifiles or some subset? Can it be hardcoded into utils?

**Comment by organisciak** [comment]
I've been thinking about this. I usually load HathiFiles with pandas and set the index to the htid. There's a convenience in in being able to call rows with meta.loc[('htid1', 'htid2', ...)]. A few deficiencies, though:

- HathiFiles are poorly formatted CSV, and it's tough (impossible?) to get a correct read on them.
- CSV is slow to read.
- Most of the info in Hathifiles isn't used most of the time.
- There's metadata that I want but can't get from the Hathifiles - particularly page count.

I wrote an initial HathiMeta class that is an accessor for the cleaned SQLite version of the Hathifiles. I figured a regular DB is more appropriate for our uses than columnar here. We can potentially improve the code for dealing with bad CSV there. I didn't add functionality for extending the metadata, but rather than overengineering I think we can just allow for extra JOIN-able tables to be imported when creating the DB for the first time.

My hope is to use this to back the HTID class and any non-json versions of the EF data. Eventually, we'll need to improve the CSV handling.
